src/slowmokit/models/bernoulliNB.hpp

```diff
@@ -0,0 +1,59 @@
+#include "core.hpp"
+using namespace std;
+
+class BernoulliNB{
+    double prior_prob(vector<int> y_train,int label){
```

Comment:
According to our guidelines variables and function names should be in camel case **not** snake case